build.sbt

```diff
@@ -40,11 +35,11 @@ lazy val core = project
     libraryDependencies ++= Seq(
       "org.typelevel" %% "cats-core" % catsV,
       "org.typelevel" %% "cats-effect" % catsEffectV,
-      "io.chrisdavenport" %% "random" % "0.0.2",
-      "io.chrisdavenport" %% "mapref" % "0.1.1",
+      "io.chrisdavenport" %% "mapref" % "0.2.1",
```

Comment:
We have `MapRef` in CE std now, I wonder if this dependency is still needed?

Reply:
I left it since there is a version of `mapref` on CE 3 from `io.chrisdavenport`, and have removed `random` since it lacks CE 3 support. But yeah, let's migrate that one too 👍🏻